meta-intel-edison-distro/recipes-core/images/edison-image.bb

```diff
@@ -81,7 +81,7 @@ IMAGE_INSTALL_append = " alsa-utils-alsamixer alsa-utils-alsactl alsa-utils-apla
 # removed python-imaging python-pygobject python-dbus 
 IMAGE_INSTALL_append = " python"
 IMAGE_INSTALL_append = " python-argparse"
-IMAGE_INSTALL_append = " python-distutils python-pkgutil python-audio python-image python-email python-netserver python-xmlrpc python-ctypes python-html python-json python-compile python-misc python-numbers python-unittest python-pydoc python-importlib"
+IMAGE_INSTALL_append = " python-distutils python-pkgutil python-audio python-image python-email python-netserver python-xmlrpc python-ctypes python-html python-json python-compile python-misc python-numbers python-unittest python-pydoc"
 
```

Comment:
I disabled importlib here, but don't know what the effect is. Also in the past removed certain python libs, that now could possibly be added back (pygobject?). I don't know python, so I have no idea what would be a sane set of libs. 

What do you think? Leave it and change when needed or try to stay as close as possible to the factory image?

Reply:
@htot here (https://lists.yoctoproject.org/pipermail/yocto/2018-May/041101.html) said "python-importlib is now part of python*-core"

Reply:
...so this specific one is clear and in general I think it doesn't really matter that much - if anyone uses a library we accidentally drop, they'll let us know. And pip can always be used to install most of them anyway after the fact, so IMHO that's not a big deal.